src/common/constants.ts

```diff
@@ -152,3 +152,15 @@ export const youtube_url = 'https://www.youtube.com/@deriv'
 export const cms_end_point = 'https://deriv-academy.directus.app'
 export const cms_assets_end_point = `${cms_end_point}/assets/`
 export const loss_percent = 70
+
+export const browsers_minimum_required_version = {
```

Comment:
these are the list of browsers used by our clients which causes the console error. I've used one or two major versions above what they use as the minimum required version for each browsers to see if it fixes the issue or not.